ansible/playbooks/AdoptOpenJDK_AIX_Playbook/main.yml

```diff
@@ -15,28 +15,35 @@
     swap_size: 4096
 
   roles:
-    - aixfs
-    - syslogs
+    # AIX base requirements
     - X11
+    - openssl
+    # verify/install IBM compilers
     - xlc_v13
     - xlc_v16
-    - openssl
+    # AIX configuration
+    - aixfs
```

Comment:
Why is this moved to after the XLC/X11 installations? Wouldn't moving this cause problems if the file systems weren't large enough to install XLC/X11? I think that's why it was at the start previously.

Reply:
This is before xlc installation - it WAS after syslogs - which is a configuration change.

What I am trying to do is get AIX BOS installation first, then AIX configuration changes (that does not require any non-BOS software) and then start looking at licensed software.

As the xlc roles use, potentially, the unarchive: module - the latest update has the role: yum before the xlc role calls.

Reply:
I agree, it looks like it is after, but it is not. No idea how the diff comes up with this.

Further - as to large enough - installp enlarges the filesystem if more space is needed.

Reply:
> What I am trying to do is get AIX BOS installation first, then AIX configuration changes (that does not require any non-BOS software) and then start looking at licensed software.

OK That reasoning is sound, but it might be worth explcitly saying that so that others modifying the file in the future adhere to the same policies

> I agree, it looks like it is after, but it is not. No idea how the diff comes up with this.

Looks ok now - it's showing where it should be based on your comments

> Further - as to large enough - installp enlarges the filesystem if more space is needed.

Yep that's fair, but I'd suggest adding that as a comment so it's clear to anyone modifying the playbook in the future